src/test/java/testresults/PostTestResultsRoadworthiness.java

```diff
@@ -0,0 +1,171 @@
+package testresults;
+
+import data.GenericData;
+import data.TestResultsData;
+import model.testresults.TestResults;
+import model.testresults.TestResultsGet;
+import model.testresults.TestResultsStatus;
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import steps.TestResultsSteps;
+import util.JsonPathAlteration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+@RunWith(SerenityRunner.class)
+public class PostTestResultsRoadworthiness {
+
+    @Steps
+    TestResultsSteps testResultsSteps;
+
+    private TestResults.Builder vehicleSubmittedData = TestResultsData.buildTestResultsSubmittedData();
+
+    private void validateSavedData(List<String> data) {
+
+        testResultsSteps.getTestResults(vehicleSubmittedData.build().getVin(), TestResultsStatus.SUBMITTED);
+        testResultsSteps.statusCodeShouldBe(200);
+        testResultsSteps.validateData((TestResultsGet) vehicleSubmittedData.build());
+        testResultsSteps.validateDataForExpiry(data);
+    }
+
+    @Title("CVSB-7675 - TC1 - AC1 - ROADWORTHINESS certificate number generated (HGV) - PASS")
+    @Test
+    public void testResults_Roadworthiness_HGV_Pass_Certificate_Number() {
+
+        // Read the base test result JSON.
+        String testResultRecord = GenericData.readJsonValueFromFile("test-results_roadworthiness_hgv_pass_7675.json", "$");
+
+        // Create alteration to add one more tech record to in the request body
+        String randomVin = GenericData.generateRandomVin();
+        String randomTestResultId = UUID.randomUUID().toString();
+        JsonPathAlteration alterationVin = new JsonPathAlteration("$.vin", randomVin, "", "REPLACE");
+        JsonPathAlteration alterationTestResultId = new JsonPathAlteration("$.testResultId", randomTestResultId, "", "REPLACE");
+        JsonPathAlteration alterationResult = new JsonPathAlteration("$.testTypes[0].testResult", "pass", "", "REPLACE");
```

Comment:
this is unnecessary since the json file already has "testResult" set as "pass"

Reply:
Changes Done.